docs/source/getting-started/first-feature.rst

```diff
@@ -129,7 +124,7 @@ This is a major benefit of writing apps on DAML: the burden of ensuring privacy
 MessageEdit Component
 ---------------------
 
-Next we need the ``MessageEdit`` component to compose and send messages to selected friends.
+Next we need the ``MessageEdit`` component to compose and send messages to selected users that we are following (and who are also following us back).
```

Comment:
We don't need to follow somebody to message them. They give us the authority to message them by following us.
```suggestion
Next we need the ``MessageEdit`` component to compose and send messages to our followers.
```